six/projects/csm/wscript

```diff
@@ -1,6 +1,6 @@
 MAINTAINER         = 'adam.sylvester@mdaus.com'
 VERSION            = '1.1.2'
-MODULE_DEPS        = 'six.sicd six.sidd'
+MODULE_DEPS        = 'scene six.sicd six.sidd'
```

Comment:
I don't think `scene` actually needs to be added here does it (this might have been something I did when I was initially hacking around)?